### ISSUE TITLE
bug: "Two-finger swipe up" does not work when VoiceOver is on in Ionic app (Accessibility)

### ISSUE THREAD

**Issue by mastergogo** [ionic-team/ionic-framework#23281]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**
"Two-finger swipe up" when voiceover is on in iOS app, should "Speak the entire screen from the top" at the moment voiceover does not read anything from the top. voiceover gesture can be found [here](https://support.apple.com/en-in/guide/iphone/iph3e2e2281/ios)

**Expected behavior:**
"Two-finger swipe up" when voiceover is on should read everything on screen from the top in iOS.

**Steps to reproduce:**
1) open the ionic app in iOS device
2) turn on VoiceOver 
3) and do a "Two-finger swipe up" 

you'll notice that, Voice over does not read anything from the top. however one thing to notice is that, when "Two-finger swipe down" voice over should "Speak the entire screen from the selected item" is working. 

Let me know if more information is needed to reproduce.

Thanks

**Comment by liamdebeasi** [comment]
Thanks for the issue! What version of iOS are you testing this on? I tested this in an Ionic app running iOS 14 and VoiceOver is reading the entire screen from the top when I do the two finger swipe up gesture.

**Comment by mastergogo** [comment]
I created a fresh ionic app to test this and I am testing in iOS 14.4.1, it is not working, however two finger down gesture is working.

**Comment by liamdebeasi** [comment]
Thanks! Are you running this in a browser or deploying with Capacitor/Cordova?

**Comment by mastergogo** [comment]
I tested after deploying the app using capacitor.

**Comment by liamdebeasi** [comment]
Thanks! I can reproduce this. This issue appears to be a bug in WebKit.

When running your app in Safari, the two finger swipe up gesture works as expected. When running the app as a PWA or inside of Capacitor/Cordova, that same gesture does not work as expected. I was able to reproduce this behavior outside of Ionic Framework.

I have reported this issue to the WebKit team: https://bugs.webkit.org/show_bug.cgi?id=225514
